javascript/javascript1/week1/goodboy-oldboy.js

```diff
@@ -0,0 +1,21 @@
+const dogYearOfBirth = 2020;
+const dogYearFuture = 2030;
+let shouldShowResultInDogYears = true;
+
+if (confirm(shouldShowResultInDogYears && "Do you want to calculate Dogyear?")) {
+    const dogYear = dogYearFuture - dogYearOfBirth;
+    console.log(
+        "Your dog will be " + dogYear + "  dog years in " + dogYearFuture
+    );
+}
+else if
+    (confirm("Do you want to calculate Dog Human Year?")) {
+    const dogHumanYear = (dogYearFuture - dogYearOfBirth) * 7;
```

Comment:
same as here.

Reply:
I tried to keep different math operation on  their respective block as per the requirement .